PeerioTechnologies/peerio-mobile
# Feat address sync

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/12673/mobile-sync-and-invite-add-contacts-ux
#### Testing instructions  
Create new account - go to contacts, should see the new zero state.
The plus button takes you to add contact.
When in the add contact screen, pressing back or the contacts tab takes you to the zero state OR to the contacts list (depends on whether you have contacts or not).
Pressing Sync address book imports any contacts you have and suggests emails from your address book that you can invite.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
